src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -247,6 +247,7 @@ uct_rc_mlx5_iface_handle_failure(uct_ib_iface_t *ib_iface, void *arg,
     }
 
     uct_rc_txqp_purge_outstanding(iface, &ep->super.txqp, ep_status, pi, 0);
+    uct_rc_ep_pending_purge_internal(&ep->super);
```

Comment:
looks like you can call something like below:
`ucs_arbiter_group_purge(&iface->tx.arbiter, &ep->arb_group, uct_rc_ep_arbiter_purge_internal_cb, NULL);`

Reply:
done